doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -17,7 +17,7 @@ command:
 include::../protos/vkGetInstanceProcAddr.txt[]
 
   * pname:instance is the instance that the function pointer will be
-    compatible with.
+    compatible with, or `NULL` for commands not dependent on any instance.
```

Comment:
BTW if I think about it, shouldn't it use the `VK_NULL_HANDLE` instead of just `NULL`?